grpc/cubic.proto

```diff
@@ -91,6 +103,26 @@ message StreamingRecognizeRequest {
 
 }
 
+// The top-level message sent by the client for the `CompileContext` request. It
+// contains a list of phrases or words, paired with a context token allowed by
+// the model being used. The token specifies a category such as "names",
+// "airports", "contacts", "product_name" etc. The context token is used to
+// determine the places in the recognition output where the provided list of
+// phrases or words may appear. The allowed context tokens for a given model can
+// be found in its ModelAttributes obtained via the `ListModels` method.
+message CompileContextRequest {
+  // Unique identifier of the model to compile the context information for.
+  string model_id = 1;
+
+  // The token that is associated with the provided list of phrases or words
+  // (e.g "names", "airports" etc.). Must be from one of the allowed tokens in
+  // the model being used.
```

Comment:
I assume these are baked into the model.  Do these values need to match the stuff in ModelInfo?  Should that relationship be documented?